testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -93,11 +93,17 @@ class ProfileTestHelper @Inject constructor(
     testCoroutineDispatchers.runCurrent()
   }
 
-  /** Login to admin profile. */
-  fun loginToAdmin() = logIntoProfile(internalProfileId = 0)
+  /** Log in to admin profile. */
+  fun logIntoAdmin() = logIntoProfile(internalProfileId = 0)
 
-  /** Login to user profile. */
-  fun loginToUser() = logIntoProfile(internalProfileId = 1)
+  /** Log in to user profile. */
+  fun logIntoUser() = logIntoProfile(internalProfileId = 1)
+
+  /**
+   * Log in to a new user profile that has no progress for any topics or stories. This relies on other
+   * tests utilizing profile 1 as the default user profile so that profile 2 never has any progress.
+   */
+  fun logIntoNewUser() = logIntoProfile(internalProfileId = 2)
```

Comment:
@BenHenning Is this correct to should this be `logInToNewUser` or `loginToNewUser` ?

Reply:
[This](https://github.com/oppia/oppia-android/pull/2246#discussion_r556981621) was the comment made -- I also had to search it and I think "log into new user" is grammatically correct usage though I can't really tell 😆

Reply:
Yeah, even I am confused in this. Thanks for clarification.